modules/core/src/main/java/org/apache/synapse/SynapseConstants.java

```diff
@@ -412,6 +412,9 @@
 
     public static final String HTTP_SENDER_STATUSCODE = "transport.http.statusCode";
 
+    public static final String HTTP_INTERNAL_SERVER_ERROR = "500";
+    public static final String HTTP_BAD_REQUEST_ERROR = "400";
+
```

Comment:
If needed I can move this to MessageProcessorConsents class as well.

Reply:
+1, let's move this to MessageProcessorConsents